### PR TITLE
Add guide for adding a custom quirk to ZHA

### PR DESCRIPTION
**Pull request by Hedda**
## Proposed change

Add a guide under ZHA troubleshooting for how users can add a custom quirk (ZHA Device Handler) for non-standard devices.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -419,6 +419,20 @@ Such a custom "ZHA Device Handler" are Python scripts that internally are also r
 
 The custom quirks implementations for zigpy implemented as ZHA Device Handlers for Home Assistant are a similar concept to that of [Hub-connected Device Handlers for the SmartThings Classics platform](https://docs.smartthings.com/en/latest/device-type-developers-guide/) as well as that of [Zigbee-Herdsman Converters (formerly Zigbee-Shepherd Converters) as used by Zigbee2mqtt](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html), meaning they are each virtual representations of a physical device that expose additional functionality that is not provided out-of-the-box by the existing integration between these platforms.
 
+#### Manually add and use a not yet merged custom ZHA quirk
+
+If you have bought a non-standard Zigbee device that is not yet supported by ZHA but someone have written a ZHA "quirk" module (ZHA Device Handler/Zigbee Converter) that not get been merged into a "[ZHA Device Handlers"](https://github.com/zigpy/zha-device-handlers)" library release then you can add that manually as a custom ZHA quirk to your Home Assistant instance, (using something like File Editor or Samba share add-ons in Home Assistant).
```
Comment (frenck) [inline source/_integrations/zha.markdown:424]: This does not belong in the end-user documentation.
We don't provide information on running custom integration or using integrations that haven't been merged yet either.

**Comment by frenck** [comment]
Thanks for the PR, but going to friendly decline it because of the above reasoning.

File: source/_integrations/zha.markdown (hunk 1 above)
Comment (Hedda) [reply source/_integrations/zha.markdown:424]: FYI, please understand that a "custom quirk" (custom ZHA Device Handler) is not the same as a "custom integration". A  a "custom quirk" is just a quirk (Zigbee converter/translator module) for a specific Zigbee device that has not yet been merged into the official ZHA Device Handlers library (a.k.a. zha-quirks), so it is part of how ZHA works.

Comment (frenck) [reply source/_integrations/zha.markdown:424]: I don't need to be taught about what they are. Thanks 👍

**Pull request closed without merging**